apps/server/src/modules/task/uc/task.authorization.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { EntityId } from '@shared/domain';
+import { CourseRepo } from '@src/repositories';
+
+export enum TaskParentPermission {
```

Comment:
rename in Permissions ?

Reply:
i thought being more specific would be good here because it is (for the moment) only intended for tasks. generally it would probably a good thing to have a centralized Permissions enum. so yes, i think we should have that soon.

Reply:
ok